run_simulator.sh

```diff
@@ -3,23 +3,37 @@
 date
 set -x
 
+echo "ROUND_START: ${ROUND_START:=1}"
+echo "ROUND_END: ${ROUND_END:=1}"
 echo "NUMBER_OF_SERVERS: ${NUMBER_OF_SERVERS:=1}"
 echo "DURATION: ${DURATION:=120}"
 echo "LOAD: ${LOAD:=$((${NUMBER_OF_SERVERS} * 80))}"
-echo "SCENARIO: ${SCENARIO}"
-RESULTS_PATH="${SCENARIO}${NUMBER_OF_SERVERS}instance(s)"
-echo "OUTPUT_PATH: ${OUTPUT_PATH:=/tmp/simulation_results/}"
 
+echo "SCENARIO: ${SCENARIO}"
```

Comment:
What is the default value for this variable? In this way you're not setting one. Is that fine if this variable is empty?

Reply:
If the user don't set some value via command line, all simulation will broke. I'll add control as a default value. It will not solve a possible user mistake since he can forget the data path too, but as we have set a default value to others,  why don't we do to this one too, right? Consider commited :) Thanks!